### DISCUSSION TITLE
將宇宙數據轉換成音檔，是否有任何科學意義?

### DISCUSSION THREAD

**Discussion by xCk27x** [1101-GS4719-C/Python-Community-GS4719-C#15] [General]
## 將宇宙數據轉換成音檔，是否有任何科學意義?
[NASA把太空中的宇宙殘骸數據資料轉換成音檔！](https://tomorrowsci.com/science/nasa%E6%8A%8A%E5%A4%AA%E7%A9%BA%E4%B8%AD%E7%9A%84%E5%AE%87%E5%AE%99%E6%AE%98%E9%AA%B8%E6%95%B8%E6%93%9A%E8%B3%87%E6%96%99%E8%BD%89%E6%8F%9B%E6%88%90%E9%9F%B3%E6%AA%94%EF%BC%81/)(資料來源)

因為本身是太空系，所以有關注一些天文相關文章
今天看到此篇報導 文中寫道
**研究人員將來自望遠鏡的數據腳本轉換成音符和節奏，由此生成音檔**
**令人驚喜的是，宇宙創作出的這些旋律非常協調，產生了許多很棒的作品**
想知道大家對此有什麼看法?

我認為這是一種非常有創意的做法，利用全新的方法挑起人們對天文的興趣，
將原本不存在(宇宙中沒有空氣)的事情用不同方法進行詮釋，在藝術方面是很好的表現。

但從科學角度來看，我們是否能從轉換後的音檔中獲得任何有用的資訊 ?

**Comment by Jie2222** [comment]
[A universe of Sound](https://chandra.si.edu/sound/)
**覺得這篇報導有相關而且也很棒!**
Whether it comes from vocal chords in our throats or the surface of the Sun, sound plays a valuable role in our understanding of the world and cosmos around us.
